ReactiveCocoa/Swift/Signal.swift

```diff
@@ -953,6 +972,18 @@ public func zip<A, B, C, D, E, F, G, H, I, J, Error>(a: Signal<A, Error>, b: Sig
 		|> map(repack)
 }
 
+/// Zips the values of all the given signals, in the manner described by
+/// `zipWith`.
+public func zip<T, Error>(signals: [Signal<T, Error>]) -> Signal<[T], Error> {
```

Comment:
This is still accepting an array (even though the implementation is now perfectly generic). Could you please change it to a sequence?